source/puppet/4.3/reference/release_notes_agent.md

```diff
@@ -30,6 +30,24 @@ The `puppet-agent` package installs Puppet 4. Also read the [Puppet 4.0 release
 
 Also of interest: [About Agent](/puppet/4.3/reference/about_agent.html) and the [Puppet 4.3 release notes](/puppet/4.3/reference/release_notes.html).
 
+## Puppet Agent 1.3.2
+
+Released December 2, 2015.
+
+This release restores a missing root certificate. No other components are updated.
```

Comment:
Right, it's correct below. Maybe something like "This release includes a more comprehensive root CA certificate bundle. No other components are updated"